tests/test_utils.py

```diff
@@ -153,9 +156,15 @@ def test_write_uvdata(save_format, tmpdir):
 @pytest.mark.parametrize("save_format", [None, "uvfits", "miriad", "uvh5", "ms"])
 def test_write_uvdata_clobber(save_format, tmpdir):
     """Test overwriting a uvdata object yields the expected results."""
+    if sys.platform.startswith("win"):
+        pytest.skip("Clobbering is flaky on windows")
+
     if save_format == "ms":
         pytest.importorskip("casacore")
 
+    if save_format == "miriad" and sys.platform.startswith("win"):
+        pytest.skip("miriad is not supported on windows")
+
```

Comment:
I think this isn't needed because the skip three lines earlier will have already skipped this test on windows.

Reply:
Good catch.